lib/index.js

```diff
@@ -14,6 +14,8 @@
       extend = require('./utils/extend'),
       isDateString = require('./utils/assert-date-string');
 
+  handleWindowResize();
```

Comment:
I think I would prefer having it in `/lib/libraries/default.js`, since this is very c3 specific.